dlme_airflow/dags/harvest_catalog.py

```diff
@@ -1,4 +1,7 @@
 from services.harvest_dag_generator import create_provider_dags, register_drivers
 
 register_drivers()
-create_provider_dags()
+
+# this comment is needed for airflow dag discovery!
```

Comment:
https://github.com/apache/airflow/blob/main/airflow/utils/file.py#L337-L357

Add a link to this in the comment.